SPECS/perl/perl.spec

```diff
@@ -1,5 +1,8 @@
 %define _unpackaged_files_terminate_build 0
 
+# RPM pipeline is rebuilding perl which gets stuck extracting debug info
```

Comment:
Not sure if we should mention infrastructure-related reasons in the spec comments. Also, shouldn't we mention this change in the changelog?

Reply:
Yes the changelog should be updated if this change stays. But for now it's a temporary change intended to unblock the build pipeline, which has been getting stuck on this package.